docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -35,7 +35,7 @@ There are three types of controls:
 For example, if you are using the *[Logs] Web Traffic* dashboard from the sample web logs data, you can add an options list for the `machine.os.keyword` field that allows you to display only the logs generated from `osx` and `ios` operating systems.
 +
 [role="screenshot"]
-image::images/dashboard_controlsOptionsList_8.6.0.png[Options list control for the `machine.os.keyword` field with the `osx` and `ios` options selected]
+image::images/dashboard_controlsOptionsList_8.7.0.png[Options list control for the `machine.os.keyword` field with the `ios` option selected]
```

Comment:
I don't think there's any options selected in the given screenshot?

![image](https://user-images.githubusercontent.com/8698078/223833265-23de441c-e874-4223-b2a3-04076b35dbc3.png)